### PR TITLE
Risk of termination of scheduled tasks 

### PR DESCRIPTION
**Pull request by huangyiminghappy**
the schedule method must catch the exception to Preventing scheduled tasks from terminating,we have  encountered similar problems

### CODE REVIEW

File: core/src/main/java/com/netflix/conductor/core/events/EventProcessor.java [hunk 1]
```diff
@@ -110,16 +112,22 @@ public EventProcessor(ExecutionService executionService, MetadataService metadat
...
+                ObservableQueue q = EventQueues.getQueue(event, false);
+                created.add(q);
+                return q;
+            }));
+            if (!created.isEmpty()) {
+                created.stream().filter(q -> q != null).forEach(queue -> listen(queue));
+            }
+        } catch (Exception e) {
+            Monitors.error(className, "event");
+            logger.error(e.getMessage(), e);
```
Comment (apanicker-nflx) [inline core/src/main/java/com/netflix/conductor/core/events/EventProcessor.java]: Please log better error message here. `logger.error` already logs the exception with its message.

Comment (huangyiminghappy) [reply core/src/main/java/com/netflix/conductor/core/events/EventProcessor.java]: ok,thanks

**Comment by huangyiminghappy** [comment]
I have remove the monitor error,thank you for reviewing

File: core/src/main/java/com/netflix/conductor/core/events/EventProcessor.java [hunk 2]
```diff
@@ -110,16 +111,21 @@ public EventProcessor(ExecutionService executionService, MetadataService metadat
...
+            events.stream().forEach(event -> queuesMap.computeIfAbsent(event, s -> {
+                ObservableQueue q = EventQueues.getQueue(event, false);
+                created.add(q);
+                return q;
+            }));
+            if (!created.isEmpty()) {
+                created.stream().filter(q -> q != null).forEach(queue -> listen(queue));
+            }
+        } catch (Exception e) {
+            logger.error(e.getMessage(), e);
```
[the remark this answers is not in the document]
Comment (huangyiminghappy) [reply core/src/main/java/com/netflix/conductor/core/events/EventProcessor.java]: ok，thanks

**Comment by huangyiminghappy** [comment]
hi ,apanicker-nflx ,can you help me review the  https://github.com/Netflix/conductor/pull/496  pull request? thank you

**Pull request merged**